jose/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "jose"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Can wait until crates are split, but we’ll need to add:
- authors
- license
- rust-version
- description
- keywords
- repository
- include ([to make sure the package is are slim as possible](https://crates.io/crates/cargo-diet))

Reply:
Agreed. Hopefully, this `Cargo.toml` goes away after we split things up.